panda/src/gobj/texturePool.h

```diff
@@ -80,18 +80,30 @@ class EXPCL_PANDA_GOBJ TexturePool {
   INLINE static const Filename &get_fake_texture_image();
   INLINE static PT(Texture) make_texture(const std::string &extension);
 
+  INLINE static bool register_filter(TexturePoolFilter *tex_filter);
+  INLINE static bool unregister_filter(TexturePoolFilter *tex_filter);
+  INLINE static void clear_filters();
+
+  INLINE static bool is_filter_registered(TexturePoolFilter *tex_filter);
+
+  INLINE static size_t get_num_filters();
+  INLINE static TexturePoolFilter *get_filter(size_t i);
```

Comment:
Let's use a MAKE_SEQ_PROPERTY interface for this (and leave the get_num_filter/get_filter unpublished, so we have only the property interface)